share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -53,7 +55,8 @@ installpkg grubby
                            --except iwlax2xx-firmware \
                            --except libertas-sd8686-firmware --except libertas-sd8787-firmware \
                            --except libertas-usb8388-firmware \
-                           --except libertas-usb8388-olpc-firmware
+                           --except libertas-usb8388-olpc-firmware \
```

Comment:
this should be libertas-firmware now (combined they were all just 512kb)

Reply:
no, the point of these lines is to exclude the *old* names. this is what I had to do to fix the tests failing on the linux-firmware updates because lorax would try to pull in *both* the old packages (from the original release repo) and the new packages (from the update), but the new packages obsolete the old ones, so it'd fail.

Reply:
ah, OK, it's hard for context sometimes in small commits :)